src/packages/emmett-mongodb/src/eventStore/mongoDBEventStore.ts

```diff
@@ -0,0 +1,272 @@
+import {
+  ExpectedVersionConflictError,
+  STREAM_DOES_NOT_EXIST,
+  assertExpectedVersionMatchesCurrent,
+  type EventStore,
+  type Event,
+  type AggregateStreamOptions,
+  type AggregateStreamResult,
+  type AppendToStreamOptions,
+  type AppendToStreamResult,
+  type ReadStreamOptions,
+  type ReadStreamResult,
+  type ReadEvent,
+  type ReadEventMetadata,
+  type ExpectedStreamVersion,
+} from '@event-driven-io/emmett';
+import { type Collection, MongoClient, type WithId } from 'mongodb';
+import { v4 as uuid } from 'uuid';
+
+export const MongoDBEventStoreDefaultStreamVersion = -1;
```

Comment:
**SUGGESTION:** If there's no particular reason to use `-1`, I'd prefer to use `0` as the default version (so just like in PostgreSQL). That's more aligned with the numbering that databases have.